[
    {
        "id": 229376107,
        "name": "django_cve_2019_19844_poc",
        "full_name": "ryu22e\/django_cve_2019_19844_poc",
        "owner": {
            "login": "ryu22e",
            "id": 301892,
            "avatar_url": "https:\/\/avatars1.githubusercontent.com\/u\/301892?v=4",
            "html_url": "https:\/\/github.com\/ryu22e"
        },
        "html_url": "https:\/\/github.com\/ryu22e\/django_cve_2019_19844_poc",
        "description": "PoC for CVE-2019-19844(https:\/\/www.djangoproject.com\/weblog\/2019\/dec\/18\/security-releases\/)",
        "fork": false,
        "created_at": "2019-12-21T04:30:34Z",
        "updated_at": "2019-12-27T06:30:25Z",
        "pushed_at": "2019-12-24T12:31:47Z",
        "stargazers_count": 71,
        "watchers_count": 71,
        "forks_count": 7,
        "forks": 7,
        "watchers": 71,
        "score": 0
    },
    {
        "id": 230083007,
        "name": "django_cve201919844",
        "full_name": "andripwn\/django_cve201919844",
        "owner": {
            "login": "andripwn",
            "id": 52893492,
            "avatar_url": "https:\/\/avatars2.githubusercontent.com\/u\/52893492?v=4",
            "html_url": "https:\/\/github.com\/andripwn"
        },
        "html_url": "https:\/\/github.com\/andripwn\/django_cve201919844",
        "description": "PoC for CVE-2019-19844 ( https:\/\/www.djangoproject.com\/weblog\/2019\/dec\/18\/security-releases\/ )",
        "fork": false,
        "created_at": "2019-12-25T10:05:48Z",
        "updated_at": "2019-12-25T10:11:41Z",
        "pushed_at": "2019-12-25T10:11:39Z",
        "stargazers_count": 1,
        "watchers_count": 1,
        "forks_count": 0,
        "forks": 0,
        "watchers": 1,
        "score": 0
    }
]